python-package/lightgbm/sklearn.py

```diff
@@ -375,7 +375,8 @@ def fit(self, X, y,
             eval_set=None, eval_names=None, eval_sample_weight=None,
             eval_class_weight=None, eval_init_score=None, eval_group=None,
             eval_metric=None, early_stopping_rounds=None, verbose=True,
-            feature_name='auto', categorical_feature='auto', callbacks=None):
+            feature_name='auto', categorical_feature='auto', 
+            init_model=None, callbacks=None):
```

Comment:
Please place the new argument either after `init_score=None` as logically correct order, or at the last place (after `callbacks=None`) as for backward compatibility.

Reply:
Done, placed it at the end for backward compatibility.